schema.json

```diff
@@ -184,12 +219,20 @@
             "type": "string"
           }
         },
+        "required": [
+          "institution",
+          "area",
```

Comment:
Why is area required? If it's a tiny area that noone knows, it might not be useful information or?

Reply:
I think without specifying the area of the study the section could not provide any useful information. For example, consider an education section witch any contains startDate and endDate of an M.S degree without specifying the area of the study. This way the degree could be in considered in computer science or the management. There is no way for the employee to know.

Reply:
Ok good point.